src/styles/block/watch.scss

```diff
@@ -0,0 +1,78 @@
+body,
+html {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+}
+
+.hand {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform-origin: 50% 100%;
+
+  &--minutes {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: rotate-minutes 3600s steps(60) infinite;
+    top: calc(50% - 20vmin);
+    margin-left: -1.5vmin;
```

Comment:
Why did you use such strange margin. You can use translate property in your keyframe transform.